tests/test_version_tag_check_action.py

```diff
@@ -97,23 +98,54 @@ def test_run_successful(mocker, version_tag, existing_tags):
 
 
 @pytest.mark.parametrize(
-    "version_tag, existing_tags, is_valid_format, is_valid_increment, expected_exit_code, error_message",
+    "version_tag, should_exist, existing_tags",
     [
-        ("invalid_version", [], False, True, 1, "Tag does not match the required format"),                                                              # Invalid format
-        ("invalid_version", [Version("v1.0.0")], False, True, 1, "Tag does not match the required format"),                                             # Invalid format
-        ("v1.0.3", [Version("v1.0.1")], True, False, 1, "New tag v1.0.3 is not one patch higher than the latest tag v1.0.1."),                          # Invalid increment
-        ("v1.0.0", [Version("v1.0.0")], True, False, 1, "The tag already exists in repository"),                                                        # Existing tag
-        ("v1.4.1", [Version("v2.0.0"), Version("v1.4.2")], True, False, 1, "New tag v1.4.1 is not one patch higher than the latest tag v1.4.2."),       # Invalid backport increment
-        ("1.0.0", [], False, True, 1, "Tag does not match the required format"),                                                                        # Invalid format and increment
-        ("v3.0.1", [Version("v2.9.9"), Version("v1.0.0")], True, False, 1, "New tag v3.0.1 is not a valid major bump. Latest version: v2.9.9."),        # Invalid version gap
+        ("v0.0.1", "true", [Version("v0.0.1")]),                             # check existence of tag - should exist
```

Comment:
In this case, if you have only one case, there is no need of parametrizing the test function.

Reply:
Yeah, I guess there will be more test cases. There is still one development issue.